components/articles/individual-article/comments/CommentsList.jsx

```diff
@@ -25,7 +25,7 @@ function CommentsList(props){
     }
     return (<section id="comments-section">
         {comments.map((comment) => {
-            return (<CommentCard key={`${comment.comment_id}-card`} comment_id={comment.comment_id} author={comment.author} body={comment.body} created_at={comment.created_at} votes={comment.votes}/>)
+            return (<CommentCard key={`${comment.comment_id}-card`} comment_id={comment.comment_id} author={comment.author} body={comment.body} created_at={comment.created_at} votes={comment.votes} setComments={setComments}/>)
```

Comment:
Spacing this out would be helpful, also just pass the comment and access the properties inside the comment card component